test/SILOptimizer/di_property_wrappers.swift

```diff
@@ -501,6 +501,26 @@ public final class Synchronized<Value> {
   }
 }
 
+struct SR_12341 {
+  @Wrapper var wrapped: Int = 10
+  var str: String
+
+  init() {
+     wrapped = 42
+     str = ""
+     wrapped = 27
```

Comment:
Can you also try assigning wrapped, wrapped then str?

Reply:
Done!